config/deb.am

```diff
@@ -38,9 +43,14 @@ if CONFIG_USER
 	pkg7=$${name}-test-$${version}.$${arch}.rpm; \
 	pkg8=$${name}-dracut-$${version}.$${arch}.rpm; \
 	pkg9=$${name}-initramfs-$${version}.$${arch}.rpm; \
+	path_prepend=`mktemp -d /tmp/intercept.XXX`; \
+	ln -s /bin/true $${path_prepend}/dh_shlibdeps; \
```

Comment:
Wait, doesn't your proposal address all of your concerns? Or is your objection more directly to not just replacing the whole `alien` kludge entirely?

My only concern was over people who might have had some of these libraries listed as "automatically installed" but this solves that problem too.